content/docs/user-guide/components/reference/terrain/world.md

```diff
@@ -34,4 +34,6 @@ We recommend that you set the **Min Height** and **Max Height** bounds to as sma
 
 For example, if a terrain uses a heightmap image with 10 texels/meter with a 1 meter height query resolution, then 9 out of 10 of the texels are ignored. If the heightmap image has 1 texel/meter with a 1 meter height query resolution, then all of the texels are used.
 
-For physics, **Height Query Resolution** controls the density of the underlying height field collider. For rendering, this determines the triangle density of the closest level of detail. **Surface Data Query Resolution** affects the density of detail materials for detail material blending.
\ No newline at end of file
+For physics, **Height Query Resolution** controls the heightfield quad spacing of the underlying collider. For rendering, this determines the triangle density of the closest level of detail.
+
+**Surface Data Query Resolution** affects the density of detail material weight data.
```

Comment:
```suggestion
**Surface Data Query Resolution** affects the density of the detail material weight data that is used for detail material blending.
```

Reply:
Nice, I like that, adds a better clarification, will update!